api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -691,11 +693,13 @@ def pick_up_tip(
                         will result in the pipette hovering over the tip but
                         not picking it up--generally not desirable, but could
                         be used for dry-run).
+                        .. deprecated:: 2.14
```

Comment:
is this correct?

Reply:
Almost but not quite. The best way to test is to build the APIv2 docs and examine them yourself, which is what I did just now.

```shell
make -C api docs-clean docs/dist/v2
```

Turns out this needs a newline. It should have (but doesn't strictly need) a deprecation message, too.

```suggestion

                        .. deprecated:: 2.14
                            Use the Opentrons App to change pipette pick-up settings.
```